### PR TITLE
Update microsoft-defender-endpoint-linux.md

### PR DESCRIPTION
**Pull request by pahuijbr**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-linux.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ In general you need to take the following steps:
   - Third-party management tools:
     - [Deploy using Puppet configuration management tool](linux-install-with-puppet.md)
     - [Deploy using Ansible configuration management tool](linux-install-with-ansible.md)
-
+    - [Deploy using Chef configuration management tool](linux-deploy-defender-for-endpoint-with-chef.md)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-linux.md:68]:
```suggestion
    - [Deploy using Chef configuration management tool](linux-deploy-defender-for-endpoint-with-chef.md)
    
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @pahuijbr, kindly commit the suggested change to insert a blank line. Many thanks.

**Review by denisebmsft — APPROVED** [review]
Thank you @pahuijbr

**Pull request merged**